js/theme/fonts.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Utility class to manage font properties to style mapping for both iOS and Android
+ *
+ * @flow
+ */
+
+import { Platform } from 'react-native'
+
+const font = 'TitilliumWeb'
+
+const fontWeights = {
+  '300': 'Light',
+  '400': 'Regular',
+  '600': 'SemiBold',
+  '700': 'Bold'
+}
+
+type FontWeight = $Keys<typeof fontWeights>
+
+type FontStyleObject = {
```

Comment:
since this type escapes the scope of this module, you want to export it

Reply:
btw I wonder if this type is already defined by react

Reply:
there is something [here](https://github.com/facebook/react-native/blob/439614891cb0ea5c0eeb6ef4d3bf1bbdec88d659/Libraries/StyleSheet/StyleSheetTypes.js#L163) - you should try to reuse what's already available if possible